src/shortcuts/Shortcut.jsx

```diff
@@ -148,6 +148,22 @@ class Shortcut extends Context {
         return this.children.length > 0;
     }
 
+    hasParentContext() { 
+        return !Lang.isEmpty(this.parentContext);
+    }
+
+    hasValueObjectAttributes() { 
+        return !Lang.isEmpty(this.valueObjectAttributes);
+    }
+
+    setAttributeIsSetByLabel(name, val) { 
+        if (!this.hasParentContext) {

```

Comment:
Is this checking for if the `hasParentContext` function is defined? Or should this be actually calling the function to get the return value

Reply:
I think it's meant to check via a function call. I can make that change and see what breaks if I change that logic.